playbooks/common/openshift-cluster/config.yml

```diff
@@ -45,6 +45,10 @@
   tags:
   - hosted
 
+- include: service_catalog.yml
```

Comment:
I think in playbooks/common/openshift-cluster/config.yml we should optionally include the playbook or not.
I think if they're calling the service catalog playbook directly that's an intentional act and we'd have to assume they want to either install, remove, or upgrade components.

I'm also fine with completely foregoing inclusion in playbooks/common/openshift-cluster/config.yml until we come out of 'tech preview'